packages/block-library/src/search/edit.js

```diff
@@ -82,6 +83,26 @@ export default function SearchEdit( {
 		style,
 	} = attributes;
 
+	const isEnhancedPagination = context?.enhancedPagination;
```

Comment:
What about the expectations for the newly introduced logic in the case when the experiment is disabled?

Reply:
oh, I see. Yes, let's add a check here to make sure that the experiment is enabled. But first let's make sure we even want to change the name of the block in the UI or have a different way to differentiate the _Instant_ Search from regular Search 🙂 .

Reply:
I've discussed it with @jasmussen who suggested NOT updating the name of the block at all and instead adding a help text:

![Screenshot 2024-12-23 at 09 47 22](https://github.com/user-attachments/assets/907645a3-b038-41f6-b4c1-6a288f0e650e)

Hence, I'll remove the updates to `search/edit.js`

Reply:
Done in 15b80fe163c33cd457ed3f69a6f6eb9c4518c064

Reply:
ok, this is a little more complicated than I had thought because there is no dedicated prop that can be passed to e.g. `<InspectorControls>` to create this help text. It's generated from the `description` field of the block's `block.json`.

I'm looking into how we can add this secondary description in the least invasive way possible.

Reply:
I see two ways we can add this:

1. As a new property in `block.json`. I don't really like that solution because it's pretty invasive for what seems like a small adjustment.
2. We could add a `<InspectorControls.Slot group="secondaryDescription">` in `block-inspector/index.js` right below the `<BlockCard>`: https://github.com/WordPress/gutenberg/blob/e5dca5461c0699b4ed902bbea375e40b0d2f803c/packages/block-editor/src/components/block-inspector/index.js#L250-L254

   This seems like a better option to me. The name of the group (`secondaryDescription`) can be adjusted of course 🙂.

Reply:
I've prototyped the second solution in https://github.com/WordPress/gutenberg/pull/68438.

I'd appreciate thoughts on whether this is a sound approach!